### PR TITLE
231 Update contributing guide with torchscript infer example

### LINKED ISSUE #231: Add an option to run inference based on torchscript weights in model.ts
Add an option to run inference based on torchscript weights model.ts

Currently it's always using model.pt weights

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes #231 .

### Description
This PR adds an instruction of using torchscript model to do validation or inference.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

I would suggest to only change the contribution guide, let's don't change the Brats bundle so far.
What do you think?

Thanks.

**Comment by yiheng-wang-nv** [comment]
> Hi @yiheng-wang-nv ,
> 
> I would suggest to only change the contribution guide, let's don't change the Brats bundle so far. What do you think?
> 
> Thanks.

Hi Nic, sure, let me revert the changes of the brats bundle.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,9 +110,15 @@ If this test is not suitable for your bundle, please add your bundle name into `
 Check the functionality of exporting the checkpoint to TorchScript file. You can also run the following command locally to verify your bundle before submitting a pull request.
 
 ```bash
-python -m monai.bundle ckpt_export --net_id network_def --filepath models/verify_model.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json
+python -m monai.bundle ckpt_export --net_id network_def --filepath models/model.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json
 ```
 
+After exporting your TorchScript file, you can check the validation or inference results based on it rather than `model.pt` with the following changes:
```
Comment (Nic-Ma) [inline CONTRIBUTING.md]: "validation" -> "evaluation".

Thanks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -110,9 +110,15 @@ If this test is not suitable for your bundle, please add your bundle name into `
...
+
+1. Remove or disable `CheckpointLoader` in validation or inference config file if exists.
+1. Define `network_def` as: `"$torch.jit.load(<your TorchScript file path>)"`.
+1. Execute validation or inference command.
```
Comment (Nic-Ma) [inline CONTRIBUTING.md]: "validation" -> "evaluation".

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**